rocky/rocky/settings.py

```diff
@@ -302,17 +296,20 @@
 
 # A list of trusted origins for unsafe requests (e.g. POST)
 # https://docs.djangoproject.com/en/4.2/ref/settings/#csrf-trusted-origins
-CSRF_TRUSTED_ORIGINS = os.getenv("DJANGO_CSRF_TRUSTED_ORIGINS", "").split()
+CSRF_TRUSTED_ORIGINS = env("DJANGO_CSRF_TRUSTED_ORIGINS", default="").split()
 
 # Configuration for Gitpod
-if GITPOD_WORKSPACE_URL := os.getenv("GITPOD_WORKSPACE_URL"):
+if GITPOD_WORKSPACE_URL := env("GITPOD_WORKSPACE_URL", default=None) is not None:
     # example environment variable: GITPOD_WORKSPACE_URL=https://minvws-nlkatcoordinatio-fykdue22b07.ws-eu98.gitpod.io
     # public url on https://8000-minvws-nlkatcoordinatio-fykdue22b07.ws-eu98.gitpod.io/
     ALLOWED_HOSTS.append("8000-" + GITPOD_WORKSPACE_URL.split("//")[1])
     CSRF_TRUSTED_ORIGINS.append(GITPOD_WORKSPACE_URL.replace("//", "//8000-"))
 
 # Configuration for GitHub Codespaces
-if GITHUB_CODESPACES_PORT_FORWARDING_DOMAIN := os.getenv("GITHUB_CODESPACES_PORT_FORWARDING_DOMAIN"):
+if (
+    GITHUB_CODESPACES_PORT_FORWARDING_DOMAIN := env("GITHUB_CODESPACES_PORT_FORWARDING_DOMAIN", default=None)
+    is not None
+):
```

Comment:
I don't understand why `is not None` is added here, because if GITHUB_CODESPACES_PORT_FORWARDING_DOMAIN would be set to the empty string we should also not added that to `ALLOWED_HOSTS`

Reply:
Good point. I wasn't sure if an empty string is valid. I'll remove it.

Reply:
It is sitll there.